.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: android
+android:
+    components:
+        # Uncomment the lines below if you want to
+        # use the latest revision of Android SDK Tools
+        - tools
+        - platform-tools
+        - tools
+
+        # The BuildTools version used by your project
+        - build-tools-27.0.3
+
+        # The SDK version used to compile your project
+        - android-27
+        - android-21
+
+        # Specify at least one system image,
+        # if you need to run emulator(s) during your tests
+        - sys-img-armeabi-v7a-android-21
+
+        # Additional components
+        - extra
+
+# Emulator Management: Create, Start and Wait
+before_script:
+    - echo no | android create avd --force -n test -t android-21 --abi armeabi-v7a
+    - emulator -avd test -no-audio -no-window &
+    - android-wait-for-emulator
+    - adb shell input keyevent 82 &
+
+before_install:
+    - yes | sdkmanager "platforms;android-27"
+
+script:
+    - ./gradlew build test connectedAndroidTest coveralls
```

Comment:
cool!

Reply:
Manually added android-27 in TravisCI because it wasn't added at the moment.